frontend/src/components/Playback/PlayerElement.vue

```diff
@@ -15,17 +15,17 @@
         :loop="playbackManager.isRepeatingOnce"
         :class="{ 'uno-object-fill': playerElement.isStretched.value }"
         @loadeddata="onLoadedData">
-        <SubtitleTrack
-          v-if="subtitleSettings.state.enabled && playerElement.currentExternalSubtitleTrack?.parsed !== undefined" />
         <track
           v-for="sub in playbackManager.currentItemVttParsedSubtitleTracks"
-          v-else
```

Comment:
This is still unclear to me, both must be rendered at the same time? To me, it makes sense for it to be one of the other, not both at the same time. Perhaps what was wrong before was the order of the conditional (so it should be track first with `!subtitleSettings.state.enabled` and SubtitleTrack afterwards with `v-else-if`?

Reply:
Because of the weird way the track element works for subtitles, loading tracks onto the media element doesn't render them, they're disabled by default actually. So they're not visible, don't load in any data, or handle any events. Until they're set to "showing" in the code.

We have to load all the tracks to be able to "show" them in the `player-element`, when available.

I don't think there's a way to use the if-else statements here because of the way tracks work.

Reply:
Alright!